pages/login.vue

```diff
@@ -75,10 +75,15 @@ export default Vue.extend({
   async beforeMount() {
     try {
       this.publicUsers = (await this.$api.user.getPublicUsers({})).data;
+    } catch (error) {
+      console.error('Unable to get public users:', error);
+    }
+
+    try {
       const brandingData = (await this.$api.branding.getBrandingOptions()).data;
       this.disclaimer = brandingData.LoginDisclaimer || '';
     } catch (error) {
-      console.error('Unable to get public users:', error);
+      console.error('Unable to get login disclaimer:', error);
```

Comment:
Would it be better to call 2 methods in to get the public users/branding data. That way we don't have to wait for a response about public users before getting branding info.

We should probably use snackbar instead of console.error

Sorry for all the seperate reviews. I've been on mobile the last few hours. Last one :D

Reply:
@Camc314 Good ideas!